nashville/scooter-data/README.md

```diff
@@ -0,0 +1,40 @@
+# Nashville Shared Urban Mobility Device (SUMD) Parking Compliance
+
+Source:
+https://data.nashville.gov/browse?tags=scooter&utf8=%E2%9C%93
+
+CSV columns:
+- 'availability_duration': Duration of the parked device's availability (HH:MM:SS)
+- 'availability_start_date': Date of the parked device's availability (YYYY-MM-DD)
+- 'availability_start_time': Time of the parked device's availability (HH:MM:SS)
```

Comment:
Are these dates and times in Nashville time?

Reply:
The availability_start_date and availability_start_time columns are in UTC